regExp1/challenges-04.test.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named isNum that takes in a string or number of any length.
+// This function should use a regular expression pattern to return true if the input
+// contains a number, and false if the input does not contain a number.
+//
+// Example input/output:
+// 12345 => true
+// '12345' => true
+// 'h3llo world' => true
+// 'hello world' => false
+// ------------------------------------------------------------------------------------------------
+
+const isNum = (num) => {
+  // Solution code here...
+  let regex = /[0-9]/g
+  if(regex.test(num)) {
+      return true;
+  }
+  else {
+      return false;
+  }
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named isCapitalized that takes in a string. This function should use a
+// regular expression pattern to match all words that begin with a capital letter.
+//
+// Return an array containing all the matches.
+// ------------------------------------------------------------------------------------------------
+
+const isCapitalized = (string) => {
+  // Solution code here...
+  let regex = /\b[A-Z]\w*/g;
+  return string.match(regex);
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 3
+//
+// Write a function named citiesAtoJ that takes in an array of city names and uses a regular expression
+// pattern to return a new array containing any cities that begin with the letters A through J, inclusive.
+// ------------------------------------------------------------------------------------------------
+
+let cities = ['Cleveland', 'San Diego', 'Birmingham', 'Seattle', 'Miami', 'New York City', 'Omaha', 'Portland', 'Austin', 'Boston', 'Newport Beach', 'Hoboken'];
+
+const citiesAtoJ = (cities) => {
+  // Solution code here...
+  let regex = /\b[A-J]\w*/g;
```

Comment:
your regex should be `/^[A-J]/` since you are just matching the beginning of your string